metadata-ingestion/src/datahub_provider/_plugin.py

```diff
@@ -275,7 +275,6 @@ def custom_on_success_callback(context):
 
 
 def task_policy(task: BaseOperator) -> None:
-    print(f"Setting task policy for Dag: {task.dag_id} Task: {task.task_id}")
```

Comment:
maybe we can change it to `logger.debug`?

Reply:
4e26016b2e0965fc390ae2760ad1f22eb78ef3a3